feat: improve `case/process` visu information (#2104)
bonitasoft/bonita-doc: modules/ROOT/pages/release-notes.adoc

== New added values

=== Bonita process monitoring now with embeded bpmn-visualization library

Bonitasoft launched the open source project https://process-analytics.dev/[Process Analytics] two years ago in response to the market need to perform BPMN process monitoring using BPMN standards.

> The deliverable of this project is called the _bpmn-visualization library_ . You can find a nice array of examples of what it can do in its https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/v0.25.2/examples/index.html[demo web site], and you can even contribute to this initiative in its https://github.com/process-analytics/bpmn-visualization-js[Github repository]. +

Reviewer: ```suggestion
The deliverable of this project is called the _bpmn-visualization library_. You can find a nice array of examples of what it can do in its https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/v0.25.2/examples/index.html[demo web site], and you can even contribute to this initiative in its https://github.com/process-analytics/bpmn-visualization-js[Github repository]. +
```

> === Bonita now embeds the bpmn-visualization library to improve process monitoring for administrators and users 

Author: This title is too long, see as it appears in the toc at the left of the page 

![image](https://user-images.githubusercontent.com/27200110/183646582-5efc65e6-447e-4df4-b1ae-27415fd721a7.png)

Author: done in e3dab0a

> Bonitasoft has launched the open source project https://process-analytics.dev/[Process Analytics] two years ago, as a response to market's need in bpmn process monitoring execution using bpmn standards. +

Author: ```suggestion
Bonitasoft launched the open source project https://process-analytics.dev/[Process Analytics] two years ago in response to the market need to perform BPMN process monitoring using BPMN standards.
```
